ui/oopsyraidsy/data/05-shb/raid/e5n.js

```diff
@@ -6,9 +6,75 @@
     ko: /^희망의 낙원 에덴: 공명편 \(1\)$/,
   },
   damageWarn: {
+    '4B8F': 'Judgment Jolt', // Stratospear explosions
+    '4E3A': 'Impact', // Stratospear landing AoE
+    '4B9C': 'Lightning Bolt', // Stormcloud standard attack
+    '4B97': 'Gallop', // Sideways add charge
+    '4BA1': 'Shock Strike', // Small AoE circles during Thunderstorm
+    '4CF2': 'Volt Strike', // Large AoE circles during Thunderstorm
+
   },
   damageFail: {
   },
   triggers: [
+    {
+      // This happens when a player gets 4+ stacks of orbs. Don't be greedy!
+      id: 'E5N Static Condensation',
+      gainsEffectRegex: gLang.kEffect.StaticCondensation,
+      mistake: function(e) {
+        return { type: 'warn', blame: e.targetName, text: e.abilityName };
+      },
+    },
+    {
+      // Helper for orb pickup failures
+      id: 'E5N Orb Tracking',
+      gainsEffectRegex: gLang.kEffect.SurgeProtection,
+      losesEffectRegex: gLang.kEffect.SurgeProtection,
+      run: function(e, data) {
+        data.hasOrb = data.hasOrb || {};
+        data.hasOrb[e.targetName] = e.gains;
+      },
+    },
+    {
+      id: 'E5N Divine Judgement Volts',
+      damageRegex: '4B9A',
+      condition: function(e, data) {
+        return !data.hasOrb;
+      },
+      mistake: function(e) {
+        return { type: 'fail', blame: e.targetName, text: e.abilityName };
```

Comment:
Do you think it'd be worth adding text more than the ability name here? like `e.abilityName + ' (no orb)'`.  Ability names on their own are usually "you got hit by something you shouldn't and everybody (technically) gets hit by this.

Reply:
That's a fair point. I'm not happy about introducing stuff like that since it requires manual translation, but you're right that otherwise it's not clear on its own.